hw03.py

```diff
@@ -0,0 +1,33 @@
+x = input("Write something:")
+if x.isdigit():
+    if int(x) % 2 == 0:
+        print("This is a number")
+        print("Number is even")
+
+    elif int(x) % 2 == 1:
```

Comment:
You can simply use `else` here since if a number is not an even number it is definitely an odd number